src/app-layout/__tests__/desktop.test.tsx

```diff
@@ -235,8 +233,15 @@ describeEachAppLayout({ sizes: ['desktop'] }, ({ theme }) => {
   test('should render badge when defined', () => {
     const { wrapper } = renderComponent(<AppLayout drawers={manyDrawers} />);
 
-    expect(isDrawerTriggerWithBadge(wrapper, manyDrawers[0].id)).toEqual(true);
-    expect(isDrawerTriggerWithBadge(wrapper, manyDrawers[1].id)).toEqual(false);
+    expect(wrapper.findDrawerTriggerById(manyDrawers[0].id, { hasBadge: true })).toBeTruthy();
+    expect(wrapper.findDrawerTriggerById(manyDrawers[1].id, { hasBadge: false })).toBeTruthy();
+  });
+
+  test('should return null when finding a non-existing drawer with hasBadge condition', () => {
```

Comment:
Minor: this test description confused me as it says "when finding..." but it doesn't find it

Reply:
Renamed to "when searching for..."